modules/apps/analytics/analytics-client-js/src/queues/baseQueue.ts

```diff
@@ -10,6 +10,7 @@ import {getRetryDelay} from '../utils/delay';
 import {getItem, setItem, verifyStorageLimitForKey} from '../utils/storage';
 
 class BaseQueue {
+	maxSize: number;
```

Comment:
Hi Didi,
this change was made because of the error shown in the test file! :relaxed: 
@interaminense